.github/workflows/enums.yml

```
@@ -0,0 +1,103 @@
+name: Custom checks
+
+on:
+  pull_request:
+    branches: [ master ]
```

```suggestion
    branches: [ master ]
    paths:
    - 'app/models/**'
```
add a path filter to run only when the content changes?

---

Great suggestion, I'll try it.

---

Works well